csm_web/scheduler/views/search.py

```diff
@@ -0,0 +1,123 @@
+from django.db.models import Count, Q
+from rest_framework import status
+from rest_framework.decorators import api_view
+from rest_framework.exceptions import PermissionDenied
+from rest_framework.response import Response
+from scheduler.models import Section, Student
+from scheduler.serializers import SectionSerializer, StudentSerializer
+
+
+@api_view(["GET"])
+def get_sections_of_user(request):
+    """
+    Gets all sections that match the queried user.
```

Comment:
I'd recommend adding a little more detail in this docstring describing exactly what the view expects from the request (ex. all of the possible query parameters, possible values for each parameter, how it handles each parameter, etc.)